packages/apps/docs/src/components/Layout/components/SideMenu/SideMenu.tsx

```diff
@@ -71,10 +72,11 @@ export const SideMenu: FC<IProps> = ({ closeMenu, menuItems }) => {
             id="search"
             onKeyUp={handleKeyPress}
             placeholder="Search"
-            isOutlined
             type="text"
             aria-label="Search"
-            endAddon={<MagnifierIcon />}
+            endAddon={
+              <MagnifierIcon className={atoms({ paddingInline: 'n2' })} />
```

Comment:
Is this padding specifically for icons? if not I would recommend adding spacing to the input itself